src/sql/workbench/contrib/notebook/browser/htmlMarkdownConverter.ts

```diff
@@ -295,3 +307,16 @@ export function addHighlightIfYellowBgExists(node, content: string): string {
 	}
 	return content;
 }
+
+export function getUriAnchorLink(node, notebookUri: URI): URI {
+	let notebookLink: URI;
+	const sectionLinkToAnotherFile = node.href.includes('#') && !node.attributes.href?.nodeValue.startsWith('#');
+	if (sectionLinkToAnotherFile) {
+		let absolutePath = !path.isAbsolute(node.attributes.href?.nodeValue) ? path.resolve(path.dirname(notebookUri.fsPath), node.attributes.href?.nodeValue) : node.attributes.href?.nodeValue;
+		// if section link is different from the current notebook
+		notebookLink = URI.file(absolutePath);
+	} else {
+		notebookLink = URI.from({ scheme: 'file', path: notebookUri.path, fragment: node.attributes.href?.nodeValue });
```

Comment:
We seem to be assuming here that these are all links to other files. What about http (or other scheme) links? e.g. `https://github.com/microsoft/azuredatastudio/blob/b743d37d6cf36f2684222d723292725ce37bbf41/src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts#L5`

Reply:
I need to add a check for this. Thank you for pointing this out 👍